<ui version="4.0" >
 <class>MvdCrewPage</class>
 <widget class="QWidget" name="MvdCrewPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="toggleLabel" >
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text" >
      <string>The view is currently showing the movie cast. Click to show directors, producers or other crew members.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="margin" >
      <number>4</number>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="1" column="0" >
    <widget class="QStackedWidget" name="stack" >
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <widget class="QWidget" name="page" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>364</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="MvdSDTreeWidget" name="cast" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>364</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="MvdSDTreeWidget" name="directors" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>364</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="MvdSDTreeWidget" name="producers" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>364</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="MvdSDTreeWidget" name="crew" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QToolButton" name="moveUpBtn" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Move the selected item up.</string>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QToolButton" name="moveDownBtn" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Move the selected item down.</string>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdSDTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>sdtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
